ui/app/controllers/evaluations/index.js

```diff
@@ -1,12 +1,83 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
+import { schedule } from '@ember/runloop';
 import { inject as service } from '@ember/service';
+import { useMachine } from 'ember-statecharts';
+import { use } from 'ember-usable';
+import evaluationsMachine from '../../machines/evaluations';
 
 export default class EvaluationsController extends Controller {
+  @service store;
   @service userSettings;
 
-  queryParams = ['nextToken', 'pageSize', 'status'];
+  @tracked width = null;
+  @tracked height = null;
+
+  @use
+  statechart = useMachine(evaluationsMachine).withConfig({
```

Comment:
I'd love to see a brief explanation to why we're using state charts for evaluations in particular. I know there are good reasons for it, but I think for our future selves' sakes, it's worth waxing on a bit.